src/Integration/LocalServices/RuleSetInspector.cs

```diff
@@ -48,15 +48,65 @@ public RuleSetInspector(IServiceProvider serviceProvider, params string[] userRu
             ruleSetSearchDirectories.Add(GetStaticAnalysisToolsDirectory());
         }
 
+        /// <summary>
+        /// <see cref="IRuleSetInspector.FindConflictingRules(string, string)"/>
+        /// </summary>
+        public RuleSet FixConflictingRules(string baselineRuleSetPath, string targetRuleSetPath)
+        {
+            if (string.IsNullOrWhiteSpace(baselineRuleSetPath))
+            {
+                throw new ArgumentNullException(nameof(baselineRuleSetPath));
+            }
+
+            if (string.IsNullOrWhiteSpace(targetRuleSetPath))
+            {
+                throw new ArgumentNullException(nameof(targetRuleSetPath));
+            }
+
+
+            RuleSet baseline = RuleSet.LoadFromFile(baselineRuleSetPath);
+            RuleSet target = RuleSet.LoadFromFile(targetRuleSetPath);
+
+            RuleConflictInfo conflicts = this.FindConlictsCore(baseline, target);
+            Debug.Assert(conflicts.HasConflicts, "There are no conflicts between baseline and target.");
```

Comment:
There might not be conflicts if the user has updated the ruleset between the calls to "find" and "fix".